modules/ROOT/pages/fd-connection-create.adoc

```diff
@@ -0,0 +1,59 @@
+= Add a Connection to the Configuration of a Connector
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+A _connection_ is a saved set of information that a running Mule app must use to connect to a specific system by means of a connector. You might create a connection as part of adding a connector to a flow.
+
+.This flowchart shows steps for adding a connector to a Mule-app project. Creating a connection can be included in those steps.
+image::fd-flowchart-configuration.png[Flowchart about adding a connector to a Mule-app project]
+
+<1> When you add a connector to a flow, you create a _configuration_, which is a set of information that includes a connection. If a configuration exists that you can use, then you can simply select it. However, if no configuration exists yet for the connector, or if no existing configuration fits your use case, then you must create a configuration.
+<2> Part of creating a configuration is specifying which connection the connector must use to connect to a system when your Mule app is running.
+<3> When the dialog for creating a configuration is opened, it presents a list of connections, if any connections already exist for this type of connector. The dialog also enables you to create a connection. For example, if you are adding a Database connector, Flow Designer opens the *Database Configuration* dialog. This dialog presents a list of connections to databases, if any connections already exist for this type of connector. The dialog also enables you to create a connection to a database.
+<4> In the procedure explained in the next section, you click the *Add connection* button to create a connection.
```

Comment:
As commented earlier, we want to avoid doc self-reference ("next section").
I'm also not sure that illustrating something prior to introducing it in narrative is a good idea.
```suggestion
<4> You can click the *Add connection* button to create a connection.
```